### PR TITLE
NIAD-1445: Update changelog

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.1.2] - 2021-05-07
 
+- Enivronment variables updated in corrisponsing vars files in docker for integration with GPCC 0.0.5 changes. (Update environment variables accordingly)
+- All vunerabile dependencies fixed with exception to one that current has no fxied list below in known issues.
```
Comment (mattd-kainos) [inline CHANGELOG.md:13]: List the fixed vulnerabilities in the "Fixed" section

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.1.2] - 2021-05-07
 
+- Enivronment variables updated in corrisponsing vars files in docker for integration with GPCC 0.0.5 changes. (Update environment variables accordingly)
```
Comment (andrewwa-kainos) [inline CHANGELOG.md:12]:
```suggestion
- Environment variables updated in corresponding vars files in docker for integration with GPCC 0.0.5 changes. (Update environment variables accordingly.)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,29 +20,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- NIAD-1398: Fix participant2 bug in allergy intolerance
+- NIAD-1410: Handle unmapped resources  with Mapper
+- NIAD-910: Map diagnostic report to compound statement 
 
 
 ## [0.1.1] - 2021-05-06
 
 ### Known Issues and Limitations
 
+- Known to be a broke version
```
Comment (andrewwa-kainos) [inline CHANGELOG.md:46]:
```suggestion
- Known to be a broken version
```

**Review by andrewwa-kainos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,72 +9,80 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.1.2] - 2021-05-07
 
+- Environment variables updated in corresponding vars files in docker for integration with GPCC 0.0.5 changes. (Update environment variables accordingly)
+- All vunerabile dependencies fixed with exception to one that current has no fxied list below in known issues.
```
Comment (andrewwa-kainos) [inline CHANGELOG.md:13]:
```suggestion
- All vulnerable dependencies fixed with exception to one that currently has no fix listed below in known issues.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -9,72 +9,80 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- NIAD-1425: Remove valueQuantity element from Blood pressure (#203)
+- NIAD-1365: Element order requeststatement (#202)
+- NIAD-1367: Correct AllergyIntolerance mapping output to match schema (#200)
+- NIAD-1380: Encode referenceRange text for XML (#199)
 
 ## [0.1.1] - 2021-05-06
 
 ### Known Issues and Limitations
 
+- Known to be a broken version due to missing environment variables needed for GPCC inegration.
```
Comment (andrewwa-kainos) [inline CHANGELOG.md:47]:
```suggestion
- Known to be a broken version due to missing environment variables needed for GPCC integration.
```

**Review by andrewwa-kainos — CHANGES_REQUESTED** [review]

**Review by andrewwa-kainos — APPROVED** [review]

**Review by mattd-kainos — APPROVED** [review]

**Pull request merged**